dna/src/commonTest/kotlin/data/DocumentAdapterTest.kt

```diff
@@ -0,0 +1,58 @@
+package com.github.trilobiitti.trilobite.dna.data
+
+import com.github.trilobiitti.trilobite.dna.testHelpers.DIAwareTest
+import kotlin.test.Test
+import kotlin.test.assertEquals
+
+class TestAdapter : DocumentAdapter() {
+    val foo: Int by this
+
+    var bar: String by this
+}
+
+class DocumentAdapterTest : DIAwareTest {
```

Comment:
This test fails on JS backend because of https://youtrack.jetbrains.com/issue/KT-22218 ...